### PR TITLE
Update lesson query forms

### PR DESCRIPTION
**Pull request by rivaquiroga**
Lesson query forms still had Anandi and María José as managing editors. 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by rivaquiroga** [comment]
@acrymble, I also edited the Managing Editor Handover section on the wiki. These files were not listed as places where name and contact of ME are available.

**Review by svmelton — DISMISSED** [review]

**Comment by rivaquiroga** [comment]
@acrymble, this is waiting for your approval

**Review by acrymble — APPROVED** [review]

**Comment by acrymble** [comment]
Sorry I didn't realise.

**Pull request merged**
